src/get-started/java-binding.md

```diff
@@ -240,6 +240,33 @@ at `/cryptocurrency/balance/:walletId`.
 See [documentation][vertx.io] on the possibilities of `Vert.x` used as a web
 framework.
 
+### Supported Services
+
+Currently Java Binding supports the following services:
```

Comment:
what does it mean "supports"? Can we call somehow Anchoring service from our service? I would add a few more sentences about these services.

Reply:
There is no public API for Anchoring service, so we cannot `call it from our service`.

Reply:
yeap. So my comment is about to add some description. Because "support" is really general here.

Reply:
I'd say "includes" some "built-in/standard" services.